annotator/static/views/rect.js

```diff
@@ -323,6 +364,10 @@ class Rect {
         return this.boundsBeforeDrag != null;
     }
 
+    isBeingDraggedViaKeydown() {
```

Comment:
Is this too much abstraction if it's used once? Maybe inline + comment would be better